crates/ruma-common/src/room_version_rules.rs

```diff
@@ -28,6 +28,9 @@ pub struct RoomVersionRules {
     /// The format of room IDs.
     pub room_id_format: RoomIdFormatVersion,
 
+    /// The format of objects referencing events.
```

Comment:
Since this is only for `auth_events` and `prev_events`, this comment would be clearer in my opinion:

```suggestion
    /// The format of arrays referencing events in PDUs.
```